### ISSUE TITLE
Collect demographic info

### ISSUE THREAD

**Issue by jyeatman** [SwipesForScience/SwipesForScience#93]
Add an easy way to collect demographic info when users sign up. For example age and highest level of education. Also, can we log the browser and device they are using?

**Comment by akeshavan** [comment]
Yes! The sign up page can be modified here: https://github.com/SwipesForScience/SwipesForScience/blob/master/src/components/SignUp.vue

**Comment by jyeatman** [comment]
Example edits to SignUp [PR#107](https://github.com/SwipesForScience/SwipesForScience/pull/107)
But I think it would be better to create a metadata form and have it pop up from the SignUp.vue just like the consent. This would allow users to configure the metadata that they want to collect and have it committed to firebase upon signup
